assets/parquet/methodology.md

```
@@ -0,0 +1,96 @@
+# Presto + Parquet(PyArrow) + Velox(Hive) methodology
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)